akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -851,6 +854,8 @@ private[projection] class R2dbcOffsetStore(
   def savePaused(paused: Boolean): Future[Done] = {
     dao
       .updateManagementState(paused, Instant.now(clock))
+      // workaround for https://github.com/r2dbc/r2dbc-mssql/pull/290
+      .map(_ => 1)
```

Comment:
do we have access to the dialect here? then we could do this conditionally only for sqlserver